### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by megan07**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
 ## 3.16.0 (Unreleased)
+FEATURES:
+* **New Data Source:** monitoring: `google_monitoring_uptime_check_ips` ([#6009](https://github.com/terraform-providers/terraform-provider-google/pull/6009))
```
Comment (danawillow) [inline CHANGELOG.md:3]:
```suggestion
* **New Data Source:** `google_monitoring_uptime_check_ips` ([#6009](https://github.com/terraform-providers/terraform-provider-google/pull/6009))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+
+IMPROVEMENTS:
+* cloudfunctions: Added `ingress_settings` field to `google_cloudfunctions_function` ([#5981](https://github.com/terraform-providers/terraform-provider-google/pull/5981))
+* cloudfunctions: added support for `vpc_connector_egress_settings` to `google_cloudfunctions_function` ([#5984](https://github.com/terraform-providers/terraform-provider-google/pull/5984))
+* cloudfunctions: added support for `vpc_connector_egress_settings` to `google_cloudfunctions_function` ([#5993](https://github.com/terraform-providers/terraform-provider-google/pull/5993))
```
Comment (danawillow) [inline CHANGELOG.md]: This is listed twice, as two different PRs.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,23 @@
...
+* accesscontextmanager: added `status.vpc_accessible_services` to `google_access_context_manager_service_perimeter` to control which services are available from the perimeter's VPC networks to the restricted Google APIs IP address range. ([#6006](https://github.com/terraform-providers/terraform-provider-google/pull/6006))
+* cloudrun: added ability to autogenerate revision name ([#5987](https://github.com/terraform-providers/terraform-provider-google/pull/5987))
+* compute: added ability to resize `google_compute_reservation` ([#5999](https://github.com/terraform-providers/terraform-provider-google/pull/5999))
+* container: added `enable_resource_consumption_metering` to `resource_usage_export_config` in `google_container_cluster` (beta only) ([#5990](https://github.com/terraform-providers/terraform-provider-google/pull/5990))
```
Comment (danawillow) [inline CHANGELOG.md:11]: this is marked beta only so probably should be removed

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,22 @@
 ## 3.16.0 (Unreleased)
+FEATURES:
+* **New Data Source:** monitoring: `google_monitoring_uptime_check_ips` ([#6009](https://github.com/terraform-providers/terraform-provider-google/pull/6009))
```
Comment (c2thorn) [inline CHANGELOG.md:3]:
```suggestion
* **New Data Source:** `google_monitoring_uptime_check_ips` ([#6009](https://github.com/terraform-providers/terraform-provider-google/pull/6009))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,21 @@
...
+* cloudfunctions: added support for `vpc_connector_egress_settings` to `google_cloudfunctions_function` ([#5984](https://github.com/terraform-providers/terraform-provider-google/pull/5984))
+* accesscontextmanager: added `status.vpc_accessible_services` to `google_access_context_manager_service_perimeter` to control which services are available from the perimeter's VPC networks to the restricted Google APIs IP address range. ([#6006](https://github.com/terraform-providers/terraform-provider-google/pull/6006))
+* cloudrun: added ability to autogenerate revision name ([#5987](https://github.com/terraform-providers/terraform-provider-google/pull/5987))
+* compute: added ability to resize `google_compute_reservation` ([#5999](https://github.com/terraform-providers/terraform-provider-google/pull/5999))
+* container: added `resource_usage_export_config` to `google_container_cluster`, previously only available in `google-beta` (ga only) ([#5990](https://github.com/terraform-providers/terraform-provider-google/pull/5990))
```
Comment (rileykarson) [inline CHANGELOG.md:11]:
```suggestion
* container: added `resource_usage_export_config` to `google_container_cluster`, previously only available in `google-beta` ([#5990](https://github.com/terraform-providers/terraform-provider-google/pull/5990))
```

**Pull request merged**